app/controllers/api/v3/notices_controller.rb

```diff
@@ -13,8 +13,9 @@ def create
     return render(status: 200, text: '') if request.method == 'OPTIONS'
 
     merged_params = params.merge(JSON.parse(request.raw_post) || {})
-    merged_params = merged_params.merge('key' => request.headers['X-Airbrake-Token']) if request.headers['X-Airbrake-Token']
-    merged_params = merged_params.merge('key' => authorization_token) if authorization_token
+    # merge makes a copy, merge! edits in place
+    merged_params = merged_params.merge!('key' => request.headers['X-Airbrake-Token']) if request.headers['X-Airbrake-Token']
+    merged_params = merged_params.merge!('key' => authorization_token) if authorization_token
```

Comment:
Hey, this all looks great @chabgood. I think in this one spot there's probably no need to assign the return value of `merge!` to `merged_params` as that's done automatically by `merge!`. Is that right?

Reply:
merged_params is just a variable above.  I edited the code above.